velox/functions/sparksql/Arithmetic.h

```diff
@@ -35,6 +35,32 @@ struct RemainderFunction {
     if (UNLIKELY(n == 0)) {
       return false;
     }
+    if constexpr (std::is_floating_point_v<TInput>) {
```

Comment:
As only few code are shared between floating point types and integral types, maybe, better to separate their implementations for `call` by using `std::enable_if_t<std::is_floating_point_v<TInput>>` to control instantiating template code differently.